segment_images.py

```diff
@@ -91,10 +138,16 @@ def main(args):
         model_type=args.sam_model_type,
         checkpoint_path=args.checkpoint,
         points_per_side=args.points_per_side,
+        points_per_batch=args.points_per_batch,
+        pred_iou_thresh=args.pred_iou_thresh,
+        stability_score_thresh=args.stability_score_thresh,
+        crop_n_layers=args.crop_n_layers,
+        crop_n_layers_downscale_factor=args.crop_n_layers_downscale_factor,
     )
 
     img_folder = Path(args.data_path, args.dataset, args.split)
 
+    # Not sure how to name the folder when there is a lot of arguments. Maybe hash?
```

Comment:
If it becomes too long I guess. Make sure to store the parameters somewhere though